cobc/flag.def

```diff
@@ -74,6 +74,10 @@ CB_FLAG_RQ (cb_max_errors, 1, "max-errors", 128, 7,
 
 /* Flags with required parameter and no associated variable */
 
+CB_FLAG_NQ (1, "format", 14,
```

Comment:
That's the correct place, guess that looks nice in the generated gnucobol pdf document (I _could_ have a look if the CI update would be live ;-). Please have a look.

The biggest issue: No one but us knows what those "magic values" mean; but luckily there's a way to fix this: https://github.com/OCamlPro/gnucobol/blob/3b31b75dc01dadfd5d4a90d3d045381e56a08fba/doc/gnucobol.texi#L406-L427 (currently missing in this PR); this _may_ could include the _facts_ about TERMINAL from non-acu, too.